ngi_pipeline/conductor/classes.py

```diff
@@ -35,7 +35,7 @@ def get_engine(self):
         try:
             return get_engine_for_bp(self.project, self.config, self.config_file_path)
```

Comment:
Now we need to figure out, why it does not find the engine (should be piper) for project P7402 - L.Dalen_17_02: https://genomics-status.scilifelab.se/project/P7402